build/wix/README.md

```diff
@@ -12,4 +12,11 @@ For example, to build 64 bits package:
 .\build.bat 64
 
 or to build 32 bits package:
-.\build.bat 32
\ No newline at end of file
+.\build.bat 32
+
+To test your multilingual package
+=================================
+
+`msiexec /i mixxx-full-name.msi ProductLanguage=xxxx`
+
+where xxxx is a language code taken from http://www.science.co.il/Language/Locale-codes.asp and available as a translation in build/wix/Localization
```

Comment:
The language codes column on http://www.science.co.il/Language/Locale-codes.asp have two chars. 
The build/wix/Localization files uses the rfc5646 notation. 
xxxx does not match any of them.